pkg/kubelet/kubelet.go

```diff
@@ -1682,6 +1684,12 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	// Fetch the pull secrets for the pod
 	pullSecrets := kl.getPullSecretsForPod(pod)
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
+		if pod.DeletionTimestamp == nil {
```

Comment:
move this condition to the same level as the if on line 1687?

Reply:
if we remove the featuregate at some-point, it is lot easier to just remove the above line, helps make review easier as well. i did this as i'm porting the same code for upstream - keeps things simple for me.

Reply:
keeping this so that feature-gate flag removal is simpler.